.github/workflows/ci.yml

```diff
@@ -140,6 +159,7 @@ jobs:
       - name: Install system dependencies (Ubuntu)
         if: runner.os == 'Linux'
         run: |
+          sudo DEBIAN_FRONTEND='noninteractive' apt-get update
```

Comment:
same as before, why changing pipeline that worked previously?

Reply:
let me remove this.